### PR TITLE
Enable it test for eventing sample

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -244,9 +244,9 @@ jobs:
           name: Eventing "Shopping Cart" sample
           command: |
             cd samples/java-eventing-shopping-cart
-            echo "Running mvn with SDK version: '$SDK_VERSION' - FIXME disabled for codegen/API work"
-            # TODO: enable integration tests with -Pit
-            # mvn -Dakkaserverless-sdk.version=$SDK_VERSION verify
+            echo "Running mvn with SDK version: '$SDK_VERSION'"
+            docker-compose -f .circleci/google-pubsub-emulator-docker-compose.yml up -d
```
Comment (patriknw) [inline .circleci/config.yml:248]: is this the wrong place for this?
there is a FIXME in `integration-tests`, but would that be active also for this step?

Comment (octonato) [reply .circleci/config.yml:248]: I don't think we need docker-compose here. The integration tests used test containers and is supposed to start whatever is needed.

Comment (patriknw) [reply .circleci/config.yml:248]: ok, then it's something missing in that integration test.

I couldn't run it locally either without starting the emulator on the side.

Comment (patriknw) [reply .circleci/config.yml]: I don't know anything about that so leaving that for someone else. Created issue https://github.com/lightbend/akkaserverless-java-sdk/issues/297

**Pull request closed without merging**